policy/release/lib/attestations.rego

```diff
@@ -41,9 +43,11 @@ tasks_from_pipelinerun := [task |
 ]
 
 # All results from the attested PipelineRun with the provided name. Results are
-# expected to contain a JSON value. The JSON value will be augmented with a key
-# "__task_name" that will hold the name of the TaskRun where the named result
-# was found.
+# expected to contain a JSON value. The return object contains the following
+# keys:
```

Comment:
Since the actual result is neatly nested under its own key now, we could do away with the `__` prefix for the keys. Maybe a further enhancement is to do away with the `key_` variables as well:
```rego
{
  "task_name": "foobar",
  "bundle": "quay.io/...",
  "value": ...,
}
```

Thoughts?